src/Business Foundation/App/NoSeries/src/Setup/NoSeriesSetupImpl.Codeunit.al

```diff
@@ -137,6 +135,23 @@ codeunit 305 "No. Series - Setup Impl."
         exit(NoSeriesSingle.MayProduceGaps());
     end;
 
+    local procedure GetNoSeriesLineFilters(var NoSeriesLine: Record "No. Series Line"; NoSeriesCode: Code[20]; StartingDate: Date)
+    var
+        NoSeries: Codeunit "No. Series";
+        NoSeriesLine2: Record "No. Series Line";
+        CodeFieldChangedErr: Label 'Change of Series Code Field is on this Record not allowed.';
+    begin
+        NoSeriesLine2.SetCurrentKey("Series Code", "Starting Date");
+        NoSeriesLine2.SetRange("Starting Date", 0D, StartingDate);
+        NoSeriesLine2.SetRange("Series Code", NoSeriesCode);
+        RaiseSetAdditionalNoSeriesLineFilters(NoSeriesLine2);
+        If NoSeriesLine2."Series Code" <> NoSeriesCode then
```

Comment:
"if NoSeriesLine2.GetFilter("Series Code") <> NoSeriesCode then" - that's what I meant. Do verify this is what GetFilter actually returns, I didn't test this completely :)